devicetypes/smartthings/arrival-sensor.src/arrival-sensor.groovy

```diff
@@ -23,9 +23,9 @@ metadata {
 		capability "Battery"
 		capability "Health Check"
 
-		fingerprint profileId: "FC01", deviceId: "019A"
-		fingerprint profileId: "FC01", deviceId: "0131", inClusters: "0000,0003", outClusters: "0003"
-		fingerprint profileId: "FC01", deviceId: "0131", inClusters: "0000", outClusters: "0006"
+		fingerprint profileId: "FC01", deviceId: "019A", deviceJoinName: "Presence Sensor"
```

Comment:
These are all SmartThings devices

Reply:
I thought so, but fingerprints have looked to be generic, so I've decided to left it out that way. Anyway, tbd.

Reply:
True, but these fingerprints have the manufacturer specific profile ID so it's highly unlikely that another device would use this DTH besides our first party stuff.